src/includes.hpp

```diff
@@ -7,9 +7,11 @@
 #endif
 
 #include <X11/Xlib.h>
+#include <fcntl.h>
```

Comment:
no need to add additional includes here - my original mistake of doing that pays now.

Move them to keybindmanager.cpp to shorten compile time.